test/js/color-handler.js

```diff
@@ -15,10 +15,10 @@ suite('color-handler', function() {
   });
   test('color interpolation', function() {
     assert.equal(webAnimations1.propertyInterpolation('color', '#00aa11', '#aa00bb')(0.2), 'rgba(34,136,51,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0), 'transparent');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'transparent', '#004488')(0.5), 'rgba(0,68,136,0.500)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(2), 'rgba(0,255,0,1)');
-    assert.equal(webAnimations1.propertyInterpolation('color', 'red', 'green')(-1), 'rgba(255,0,0,1)');
+    assert.equal(webAnimations1.propertyInterpolation('fill', 'transparent', '#004488')(0), 'transparent');
+    assert.equal(webAnimations1.propertyInterpolation('stroke', 'transparent', '#004488')(0.5), 'rgba(0,68,136,0.500)');
+    assert.equal(webAnimations1.propertyInterpolation('lightingColor', 'red', 'green')(2), 'rgba(0,255,0,1)');
+    assert.equal(webAnimations1.propertyInterpolation('textDecorationColor', 'red', 'green')(-1), 'rgba(255,0,0,1)');
```

Comment:
Do we need to test all the different properties? If so, we seem to be missing some.